canonical-web-and-design/maas-ui
# Sync legacy js files to latest launchpad master branch.

## Done
- Added new changes in launchpad master branch, up to `dc3fd1c40 Show interface members (read-only) in the Network table.` ([Commit in lp](https://git.launchpad.net/maas/commit/?id=dc3fd1c4091e2eda057bd1d9997551e9f052efee))
- Drive-by fixes:
  - Machine summary cards get parent classes correctly
  - Pod details template loaded correctly

## QA
- Check that the machine details pages looks like the current launchpad master (summary, network, storage tabs)
- Check that the pod details summary loads in correctly
- Really the only changes that haven't already been merged in master and need code review are in:
  - `legacy/src/app/entry.js`
  - `legacy/src/app/directives/pod-details/summary.js`
  - `legacy/src/app/partials/pod-details.html`
  - `legacy/src/app/partials/nodedetails/node-details-summary.html`
